pkg/model/resource/options_test.go

```diff
@@ -47,27 +47,27 @@ var _ = Describe("Resource Options", func() {
 			options := &Options{Group: "crew", Version: "1", Kind: "FirstMate"}
 			Expect(options.Validate()).NotTo(Succeed())
 			Expect(options.Validate().Error()).To(ContainSubstring(
-				`version must match ^v\d+(alpha\d+|beta\d+)?$ (was 1)`))
+				`version must match ^v\d+(?:alpha\d+|beta\d+)?$ (was 1)`))
```

Comment:
IMO : it should be a const

Reply:
I also thought of that to prevent having it hardcoded a doxen times the thing is that the string is not exported from its package. I think the better approach would be to create an expecific error instead a generic one and test if this yields this kind of error. But I just wanted to do the less changes to get it to pass Travis for now.